### PR TITLE
Extraction of chain configuration

### LINKED ISSUE #89: refactor chain configuration parameters and logic to independent files
Collecting varying iterations of chain parameters adjacent one another (line over line) causes merge complexity. 

This issue proposing, where possible, to refactor chain-specific parameters and logic to own-file patterns, thereby easing this workflow.

This would look something like:

Before:
```go
// params/chain_ids.go
const ChainAID = 1
const ChainBID = 2
```

After:
```go
// params/chain_ids_a.go
const ChainAID = 1
```

```go
// params/chain_ids_b.go
const ChainBID = 2
```

### PR DESCRIPTION
**Pull request by tzdybal**
This will resolve #89.

### CODE REVIEW

**Comment by tzdybal** [comment]
@meowsbits if you know any other place when I should extract something, please let me know.
And take a look at fcae945e1c947adea5b8aa6f8655b78fde08b3c7 - if found that some chains are missing - or this is intentional omission?

**Comment by meowsbits** [comment]
https://github.com/multi-geth/multi-geth/pull/92/commits/45036af45caada98a281da15855ce4abda89efef / https://github.com/multi-geth/multi-geth/commit/fcae945e1c947adea5b8aa6f8655b78fde08b3c7 LGTM; I think that must have been an accidental omission. 

Interestingly, upstream seems to have same issue: https://github.com/ethereum/go-ethereum/blob/master/core/genesis.go#L230-L241.

File: params/config_classic.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		Ethash:              new(EthashConfig),
+	}
+
+	DisinflationRateQuotient = big.NewInt(4)      // Disinflation rate quotient for ECIP1017
+	DisinflationRateDivisor  = big.NewInt(5)      // Disinflation rate divisor for ECIP1017
+	ExpDiffPeriod            = big.NewInt(100000) // Exponential diff period for diff bomb & ECIP1010
+
+	// ClassicBootnodes are the enode URLs of the P2P bootstrap nodes running on
+	// the Ethereum Classic network.
+	ClassicBootnodes = []string{
```
Comment (meowsbits) [inline params/config_classic.go]: Perhaps `*Bootnodes` vars should be in `params/bootnodes_*.go` files? (Just to echo the existing pattern).

Comment (tzdybal) [reply params/config_classic.go]: I was thinking about this, but then I thought that having separate file for a just 1 or 2 `var`s seems to be an overkill. Same for `genesis_alloc`.

Comment (meowsbits) [reply params/config_classic.go]: Roger, understood. I guess despite the overkill I'd prefer to see as close to a 'reflective iteration' of upstream patterns whenever possible just to emphasize structural consistency (as opposed to introducing sane arbitrations ;)  Also then I don't have to remember anything new when I try to look up Kotti bootnodes )

Not fervent or evangelical in my view though, just sayin'. LGTM as-is, too.

**Review by meowsbits — APPROVED** [review]

File: params/config_classic.go (hunk 1 above)
Comment (meowsbits) [reply params/config_classic.go]: Ha, on the flipside, though, files like this https://github.com/multi-geth/multi-geth/pull/88/files#diff-585fd2d517a40a26f4efdec5e4edf4ca will really push the fundamentalist overkiller envelope.

Comment (tzdybal) [reply params/config_classic.go]: 10 consts - seems legit :rofl:

**Comment by tzdybal** [comment]
Force push caused by rebase to multi-geth/multi-geth master.

**Comment by tzdybal** [comment]
@meowsbits I had to add missing Kotti bootnodes, and yes. It's much more "natural" to follow the existing pattern, instead of searching for right file. I'll split the configs.

**Comment by tzdybal** [comment]
@meowsbits please re-review.

**Review by meowsbits — APPROVED** [review]

**Comment by tzdybal** [comment]
Can anyone press the green button? ;)

**Pull request merged**